Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/braintree/braintree_ios",
         "state": {
           "branch": null,
-          "revision": "6be5b6ceabe6acf3353a986da272bc604c8cd010",
-          "version": "5.4.2"
+          "revision": "914f9c60232d599ed61d52be1ece34b7e700faf6",
+          "version": "5.8.0"
```

Comment:
Weird that this was 5.4.2 before

Reply:
Yeah, it actually didn't update/resolve correctly when we made the version change either. We had to try some manual commands to get it to the right version. May want to keep an eye on this in the future.